api/src/formatters/strings.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { loadFixture, simpleFormatFixture } from "./fixtures";
+import { stringsParser, stringsExporter } from "./strings";
+import { async } from "rxjs/internal/scheduler/async";
```

Comment:
Could you remove this import? The editor probably added it automatically.

Reply:
rebased